src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx

```diff
@@ -41,10 +45,44 @@ export const RightsActionBarContent = ({
   serviceDescription,
   rightDescription,
   serviceIdentifier,
-  errorCode,
 }: RightsActionBarContentProps) => {
   const { t } = useTranslation('common');
   const hasUndelegableRights = rights.some((r) => r.delegable === false);
+  const [actionList, setActionList] = useState<string[]>([]);
+  const [errorList, setErrorList] = useState<string[]>([]);
+
+  const createErrorList = () => {
+    const errors: string[] = [];
+    rights.map((right: Right) => {
+      if (right.delegable === false) {
+        right.details?.map((detail) => {
```

Comment:
It isn't clear to me how you're thinking we're going to push to errors when doing it like you suggest. Can you propose the full code as suggestion?

Reply:
No, you're right, I didn't register that this was a double loop used to create an unrelated list.
But since you're not actually mapping anything, maybe a double for loop would be more appropriate?

The use of map makes me expect that the result will have the same number of elements as the array being mapped

Reply:
Good point